modules/food-menu/food-item-detail.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+
+import * as React from 'react'
+import {Text, ScrollView, View, StyleSheet} from 'react-native'
+import {Column, Row} from '@frogpond/layout'
+import {ListRow, ListSeparator, Detail, Title} from '@frogpond/lists'
+import * as c from '@frogpond/colors'
+import map from 'lodash/map'
+import {DietaryTags} from './dietary-tags'
+import {type TopLevelViewPropsType} from '../../source/views/types'
+import type {
+	MenuItemType as MenuItem,
+	NutritionDetailType,
+	MasterCorIconMapType,
+} from './types'
+
+type Props = TopLevelViewPropsType & {
+	navigation: {state: {params: {icons: MasterCorIconMapType, item: MenuItem}}},
+}
+
+export class MenuItemDetailView extends React.Component<Props> {
+	static navigationOptions = {
+		title: 'Nutrition',
+	}
+
+	render() {
+		const item = this.props.navigation.state.params.item
```

Comment:
Use object destructuring here, please – `const {item, icons} = this.props.navigation.state.params`